tests/dummy/app/templates/index.hbs

```diff
@@ -1,36 +1,716 @@
-<DocsHero
-  @logo="ember"
-  @slimHeading="My"
-  @strongHeading="Addon"
-  @byline="Statecharts for Ember.js applications powered by XState"
-/>
-<div class="docs-container">
-  <div class="docs--m-8 docs-py-16 docs-text-center">
-    <div>
-      <div class="aspect-ratio aspect-ratio--16x9">
-        <iframe
-          class="aspect-ratio--object"
-          width="640"
-          height="360"
-          title="Constructing Web Apps With Statecharts"
-          src="https://www.youtube.com/embed/g2GegNGgW84?start=17"
-          frameborder="0"
-          allow="autoplay; encrypted-media"
-          allowfullscreen
+<div class='bg-white'>
+  <main>
+    <div class='pt-10 bg-gray-900 sm:pt-16 lg:pt-8 lg:pb-14 lg:overflow-hidden'>
+      <div class='mx-auto max-w-7xl lg:px-8'>
+        <div class='lg:grid lg:grid-cols-2 lg:gap-8'>
+          <div
+            class='max-w-md px-4 mx-auto sm:max-w-2xl sm:px-6 sm:text-center lg:px-0 lg:text-left lg:flex lg:items-center'
+          >
+            <div class='lg:py-24'>
+              <h1
+                class='mt-4 text-4xl font-extrabold tracking-tight text-white sm:mt-5 sm:text-6xl lg:mt-6 xl:text-6xl'
+              >
+                <span class='block'>
+                  A better way to
+                </span>
+                <span
+                  class='block pb-3 text-transparent bg-clip-text bg-gradient-to-r from-ember to-red-50 sm:pb-5'
+                >
+                  build Ember.js apps
+                </span>
+              </h1>
+              <p
+                class='text-base text-gray-300 sm:text-xl lg:text-lg xl:text-xl'
+              >
+                <strong class='align-middle'>
+                  ember-statecharts
+                </strong>
+                <span class='align-middle'>
+                  provides an easy way to use
+                </span>
+                <a
+                  href='https://xstate.js.org/'
+                  target='_blank'
+                  rel='noopener noreferrer'
+                >
+                  {{svg-jar
+                    'xstate'
+                    class='inline w-16 h-8 align-middle fill-current text-current hover:text-white'
+                  }}
+                </a>
+                <strong class='align-middle'>
+                  statecharts
+                </strong>
+                <span class='align-middle'>
+                  in Ember.js applications
+                </span>
+              </p>
+              <div class='mt-4 sm:mt-6'>
+                <div class='items-center sm:flex'>
+                  <div class='flex-1 min-w-0'>
+                    <div
+                      class='px-4 py-2 my-6 overflow-hidden bg-gray-100 rounded-md'
+                    >
+                      <code>
```

Comment:
We could add [`.select-all`](https://tailwindcss.com/docs/user-select#select-all-text-in-one-click) so it's selected in one click...